draft-ietf-oauth-sd-jwt-vc.md

```diff
@@ -1129,6 +1360,10 @@ After the validation, the Verifier will have the following data for further proc
 
 <{{examples/03-pid/verified_contents.json}}
 
+## Example 2: Type Metadata {#ExampleTypeMetadata}
+
+<{{examples/typemetadata/example.json}}
```

Comment:
I know there's a certain appeal to this kind of file include in the abstract but IMHO hard won experience with the technique has shown that it makes maintenance, contributions, and general progress on the whole document project much more difficult and error prone. Unless there's compelling reason to have a separate file (e.e., automated generation or preprocessing or similar), I will *highly* suggest we just inline these things.